next-sitemap.config.js

```diff
@@ -49,6 +49,10 @@ module.exports = {
         disallow: [
           "/404",
           "/500",
+          "/*Browse.aspx$",
+          "/*browse.aspx$",
```

Comment:
Check if the other URLs in the list would also need * and/or $

Reply:
For these 2 routes in particular I am using the `$` because I only want to exclude routes ending with `browse.aspx`


According to the app insights logs were not getting any requests that *contain* `browse.aspx` unless the *end with* it


<img width="661" alt="image" src="https://github.com/user-attachments/assets/0c7531f2-c4d9-490c-882d-7cbeb22957f4">

Reply:
I did remove 
These routes because the wildcard routes already cover both edge cases
`/*BrowseXML.aspx$`
`/*browsexml.aspx$`